static/js/ui_init.js

```diff
@@ -129,6 +130,15 @@ function message_hover(message_row) {
     message_row.find(".edit_content").html(render_edit_content_button(args));
 }
 
+function initialize_navbar() {
+    const rendered_navbar = render_navbar({
+        embedded: page_params.narrow_stream !== undefined,
```

Comment:
This statement is actually a replica of: https://github.com/zulip/zulip/blob/0740c94517b45f4523d297a1932da9ab7daa5886/zerver/views/home.py#L215 

As `narrow_stream` can be present or absent in `page_params` dict.
https://github.com/zulip/zulip/blob/0740c94517b45f4523d297a1932da9ab7daa5886/zerver/lib/home.py#L225-L234